source/ui/Image.js

```diff
@@ -174,12 +186,17 @@
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
 			if (this.sizing) {
-				this.applyStyle('background-image', src ? 'url(' + enyo.path.rewrite(src) + ')' : 'none');
```

Comment:
Removing 'none' removes the ability for one to "unset" the background image. This would mean that the property would be removed and whatever the property inherited would take prescience, so it would be impossible to take away the image, and only restore it to what some CSS had applied earlier.